src/Pages/Repositories/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -468,302 +373,261 @@ const AddContent = () => {
-                        variant='link'
-                        icon={<MinusCircleIcon />}
-                      >
-                        Remove
-                      </Button>
-                    </Hide>
-                  </Td>
-                </Tr>
+      {isEdit && isLoadingInitialContent ? (
+        <Loader />
+      ) : (
+        <Form>
+          <FormGroup label='Name' isRequired fieldId='namegroup'>
+            <TextInput
+              isRequired
+              id='name'
+              name='name'
+              label='Name'
+              ouiaId='input_name'
+              type='text'
+              validated={getFieldValidation('name')}
+              onChange={(_event, value) => {
+                updateVariable({ name: value });
+              }}
+              value={name || ''}
+              placeholder='Enter name'
+            />
+            <CustomHelperText
+              hide={getFieldValidation('name') === 'default'}
+              textValue={errors?.name}
+            />
+          </FormGroup>
+          <FormGroup
+            label='Repository type'
+            fieldId='repositoryType'
+            hasNoPaddingTop
+            // labelIcon={
+            //   <Tooltip content='Put important explanation in here: Put important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in here'>
+            //     <OutlinedQuestionCircleIcon className='pf-u-ml-xs' color={global_Color_200.value} />
+            //   </Tooltip>
+            // }
+          >
+            <Flex direction={{ default: 'column' }} gap={{ default: 'gap' }}>
+              <Hide hide={isEdit && contentOrigin === ContentOrigin.UPLOAD}>
+                <Radio
+                  isChecked={values.snapshot && values.origin === ContentOrigin.EXTERNAL}
+                  id='snapshot_radio'
+                  label='Snapshotting'
+                  description={
+                    values.snapshot && values.origin === ContentOrigin.EXTERNAL
+                      ? 'Enable snapshotting for an external repository, allowing you to build images with historical snapshots.'
+                      : ''
+                  }
+                  name='snapshot-radio'
+                  onClick={() =>
+                    setValues({ ...values, snapshot: true, origin: ContentOrigin.EXTERNAL })
+                  }
+                />
+                <Radio
+                  isChecked={!values.snapshot && values.origin === ContentOrigin.EXTERNAL}
+                  id='introspect_radio'
+                  label='Introspect only'
+                  description={
+                    !values.snapshot && values.origin === ContentOrigin.EXTERNAL
+                      ? 'Enable only introspection for an external repository, snapshots will not be taken.'
```

Comment:
Enable only introspection for an external repository, for building images against the external repository.